### PR TITLE
Walls and Custom Movement

### PR DESCRIPTION
**Pull request by Rohanator9000**

### CODE REVIEW

File: custom-movement.js [hunk 1]
```diff
@@ -0,0 +1,388 @@
+import { defs, tiny } from "./external/common.js";
```
Comment (haleykim227) [inline custom-movement.js:1]: I'm assuming custom-movement.js defines all the key-press movements we talked abt, and is called by wonderland.js which is then called by main-scene.js? just making sure i get the flow correctly

File: external/obj-file-demo.js [hunk 2]
```diff
@@ -0,0 +1,156 @@
+import {defs, tiny} from './common.js';
```
Comment (haleykim227) [inline external/obj-file-demo.js:1]: did u get obj-file-demo.js from somewhere? asking bc of the nicely written paragraph style comments like in our assignment code.

also, where is this actually called?

File: custom-movement.js (hunk 1 above)
Comment (Rohanator9000) [reply custom-movement.js:1]: Yes!

File: external/obj-file-demo.js (hunk 2 above)
Comment (Rohanator9000) [reply external/obj-file-demo.js:1]: Yeah, everything in `/external/` is copied from the assignment files. There may be a few edits for compatibility.

I use `Shape_From_File` in `wonderland.js` to load `.obj` files.

**Pull request merged**